pulseapi/entries/serializers/project.py

```diff
@@ -0,0 +1,42 @@
+from ..models import Entry
+from .base import EntrySerializerWithCreators
+
+
+class ProjectEntrySerializer(EntrySerializerWithCreators):
+    """
+    Serializes an entry based on the fact that we
+    know that it is a project. which means:
+
+    required:
+        - id,
+        - author
+        - title
+
+    useful:
+        - description
+        - url
+        - issue categories
+        - tags
+        - contributors
+        - get involved
+        - get involved url
+        - "is WIP?"
+        - help categories
+        - thumbnail image
+
+    extraneous
+        - custom text
+        - custom selector
+
+    This is, effectively, the same as a base entry.
```

Comment:
If this is going to be a read/write serializer, we probably should:
1. set `fields` to include the required and useful fields
2. explicitly declare overrides to the fields in `EntrySerializerWithCreators`. For example, `content_url` is not required here but is for regular entries (we'll need to change the model to reflect that btw, coz currently it's not nullable but it should be) -- so in the base serializer we would explicitly declare `content_url = serializers.URLField(required=True)` and in this serializer we would declare `content_url = serializers.URLField()`.